test/performance/latency-controller/README.md

```diff
@@ -1,242 +1,159 @@
+4. **🏭 Activation → Workload Object Created (WEC)**
+   - The workload object (e.g., Deployment, Service) is created in the WEC cluster.
 
-```bash
-# Deployment 1
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: nginx-singleton-deployment
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: nginx
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 80
-          initialDelaySeconds: 5
-          periodSeconds: 5
-EOF
-```
+5. **📝 Status Collection → WorkStatus Created (ITS)**
+   - The WEC cluster reports status back to the ITS cluster through a `WorkStatus` object.
 
-```bash
-# Deployment 2 (variation)
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: apps/v1
-kind: Deployment
-metadata:
-  name: nginx-test-deployment
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  replicas: 1
-  selector:
-    matchLabels:
-      app: nginx
-  template:
-    metadata:
-      labels:
-        app: nginx
-    spec:
-      containers:
-      - name: nginx
-        image: nginx:alpine
-        ports:
-        - containerPort: 80
-        readinessProbe:
-          httpGet:
-            path: /
-            port: 80
-          initialDelaySeconds: 5
-          periodSeconds: 5
-EOF
-```
+6. **✅ Status Propagation → WDS Object Status Updated**
+   - The collected status is propagated back and reflected in the original WDS object.
 
-### Pods
+## Features
 
-Apply pods to generate observable latency metrics:
+### Advanced Monitoring Capabilities
 
-```bash
-# Pod 1
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: v1
-kind: Pod
-metadata:
-  name: nginx-pod
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  containers:
-  - name: nginx
-    image: nginx:alpine
-    ports:
-    - containerPort: 80
-    readinessProbe:
-      httpGet:
-        path: /
-        port: 80
-      initialDelaySeconds: 5
-      periodSeconds: 5
-EOF
-```
-
-```bash
-# Pod 2 (variation)
-kubectl --context "$wds_context" apply -f - <<EOF
-apiVersion: v1
-kind: Pod
-metadata:
-  name: nginx-pod-2
-  namespace: default
-  labels:
-    app.kubernetes.io/name: nginx-singleton
-spec:
-  containers:
-  - name: nginx
-    image: nginx:alpine
-    ports:
-    - containerPort: 80
-    readinessProbe:
-      httpGet:
-        path: /
-        port: 80
-      initialDelaySeconds: 5
-      periodSeconds: 5
-EOF
-```
+- **Multi-Cluster Support**: Monitors workloads across multiple WEC clusters simultaneously  
+- **Dynamic Resource Discovery**: Automatically discovers and monitors all configured Kubernetes resource types  
```

Comment:
@asmit27rai, we need to clarify there that only `namespaced` scope resources works. This controller for now only convers namespaced scope resources.